pkgs/applications/networking/owncloud-client/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   name = "owncloud-client-${version}";
-  version = "2.3.4";
+  version = "2.4.1";
 
   src = fetchurl {
     url = "https://download.owncloud.com/desktop/stable/owncloudclient-${version}.tar.xz";
-    sha256 = "1fpi1mlp2b8sx2993b4mava5c6qw794dmlayih430299z1l9wh49";
+    sha256 = "4462ae581c281123dc62f3604f1aa54c8f4a60cd8157b982e2d76faac0f7aa23";
   };
 
-  patches = [ ../nextcloud-client/find-sql.patch ];
+  patches = [ ./find-sql.patch ];
```

Comment:
Would you mind opening a pull request in the [upstream repo](https://github.com/owncloud/client) and adding a link here as a comment?

Reply:
Well... I made the version bump based on the 2.3.4 version currently in 18.03 stable. This is based on the Enterprise Version from owncloud.com, but there is no way to pull requests. The opensource Standard Edition is the one you mention as upstream repo but it has a different structure and the patch above does not apply. Should we switch to the Standard Edition? (It seems the Enterprise Version has more features, but I don't really care as I'm using only basic functionalities). Maybe @qknight has a take on this?